setup.cfg

```diff
@@ -14,9 +14,16 @@ packages = find:
 include_package_data = True
 python_requires = >=3.7
 setup_requires = setuptools_scm
-install_requires = 
+install_requires =
+  datetime
+  dask
+  gcsfs
+  numpy
+  pandas
   torch>=1.8.0
```

Comment:
double check that these are all that are required to install the package in a fresh environment